### PR TITLE
fix(payment): INT-4698 VISA Checkout load payment information

### PR DESCRIPTION
**Pull request by BC-PabloRivera**
## What? [INT-4698](https://jira.bigcommerce.com/browse/INT-4698)
Using postForm to checkout.php to get the customer mail and shipping data. I choose to do it this way because it's the same thing google pay strategy does to get the data

## Why?
Customers have reported that when using Visa Checkout they are routed back to the checkout with the billing address completed but cannot make progress until refreshing the page.

## Testing / Proof
After Visa Checkout before:
image

After Visa Checkout now: 
image

@bigcommerce/checkout @bigcommerce/payments

### CODE REVIEW

**Review by JoseAlvarezHdz — APPROVED** [review]
LGTM 👍🏾

**Comment by JelopezBC** [comment]
💚 Tested!
PR has been validated and is working as expected.

Following scenarios has been validated:

- Orders with Auth Only 
- Orders with Auth & Capture 
- Void orders
- complete Visa checkout orders.
- Different credit cards (VISA, MC, etc).
- etc.

Please review attached document with validation details.
[INT-4698 shelfology.com - Visa Checkout is not processing initial payment request.docx](https://github.com/bigcommerce/checkout-sdk-js/files/7140001/INT-4698.shelfology.com.-.Visa.Checkout.is.not.processing.initial.payment.request.docx)

File: src/customer/strategies/braintree/braintree-visacheckout-customer-strategy.ts [hunk 1]
```diff
@@ -121,10 +124,22 @@ export default class BraintreeVisaCheckoutCustomerStrategy implements CustomerSt
                     state.shippingAddress.getShippingAddress(),
                     state.billingAddress.getBillingAddress()
                 )
-                .then(() => this._store.dispatch(this._checkoutActionCreator.loadCurrentCheckout()));
+                .then(() => {
+                    this._store.dispatch(this._checkoutActionCreator.loadCurrentCheckout());
```
Comment (bc-mackxu) [inline src/customer/strategies/braintree/braintree-visacheckout-customer-strategy.ts:128]: Could we add async like below?  
```suggestion
                .then(async () => {
                    await this._store.dispatch(this._checkoutActionCreator.loadCurrentCheckout());
```

**Comment by BC-PabloRivera** [comment]
♻️ @bc-mackxu

**Review by animesh1987 — APPROVED** [review]

**Comment by JelopezBC** [comment]
💚 Tested!

PR has been validated and is working as expected.

Following scenarios has been validated:

Orders with Auth Only 
Orders with Auth & Capture 
Void orders
complete Visa checkout orders.
etc.
Please review attached document with validation details.

[INT-4698 Visa Checkout is not processing initial payment request V2.docx](https://github.com/bigcommerce/checkout-sdk-js/files/7237045/INT-4698.Visa.Checkout.is.not.processing.initial.payment.request.V2.docx)

**Pull request merged**